doc/specifications/identities/Remote-identities-creation-diagram.puml

```diff
@@ -0,0 +1,58 @@
+@startuml
+actor "Authenticated user" as user
+participant "Client" as client
+database "Backend" as backend
+user -->o client: click\n« add remote account »
+client -> backend: ""GET /providers""
+backend -> client: providers list\nwith additional data\nto pre-fill form
+hnote left client: UI : « choose a provider »
+note over user
+User
+1. select a provider
+2. fill remote address, name, etc.
+3. fill connection settings (if required)
+         (for gmail and twitter,
+         server settings are pre-filled)
+4. if applicable :
+        select auth method : password or Oauth
+5. if applicable : fill credentials
+        (for ex. password auth)
+end note
+user -->o client: save form
+client -> backend: ""POST /identities/remotes""
+group Oauth mechanisms
+group Oauth1 only (Twitter for ex.)
+create boundary provider
+backend -> provider: request auth parameters
+provider -> backend: auth params
+end
+group Oauth2 only (Gmail for ex.)
+backend --> backend: retrieve application secret params\nfrom backend
+end
+backend -> client: ""202 Accepted""\nwith Oauth parameters in payload
+hnote left client: UI : « you must authorize Caliopen »
+client -> user: UI for Oauth redirect
```

Comment:
```
client -> user: open popup with Oauth redirect
```

this means this is a brand new window with it's own context.
So the client has no clue when the process is finished (nor if it is successful).